app/views/layouts/volunteer_registration.html.haml

```diff
@@ -0,0 +1,35 @@
+!!!
+%html{:xmlns => "http://www.w3.org/1999/xhtml"}
+  %head
+    %meta{:content => "width=device-width", :name => "viewport"}/
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %title Spectrum Trail Racing Confirmation Email
+    %link{rel: :stylesheet, href: "email.css"}
+
+  %body{:bgcolor => "#FFFFFF"}
+    %table.head-wrap{:bgcolor => "#182B3F"}
+      %tr
+        %td
+        %td.header.container
+          .content
+            %table{:bgcolor => "#182B3F"}
+              %tr
+                %td
+                  %a{href: root_url}
+                    %img{:src => asset_path('logo-text-inside.png'), style: "max-width: 100px;"}/
```

Comment:
I had to change  ```asset_url``` to  ```asset_path``` to get the preview to render, however The image doesn't render on the local letter_opener email. I need to research the differences to understand which needs to be included for production.

Reply:
@ColinW520 After doing some research it looks like ```asset_path``` is used when you have the asset included on the server hosting the code base and ```asset_url``` when you're using a CDN or the asset is hosted elsewhere. Seems like maybe ```asset_url``` with no options passed used to behave like ```asset_path``` before it was introduced. Just want to confirm it's ok for me to use ```asset_path``` here.